packages/sdk/src/instant-trade/InstantTradeSellerTxBuilder.ts

```diff
@@ -51,14 +51,16 @@ export default class InstantTradeSellerTxBuilder extends InstantTradeBuilder {
   }
 
   private async generateSellerOutputs() {
-    const royalty = await this.calculateRoyalty()
-    const royaltyAmount = royalty && royalty.amount >= MINIMUM_AMOUNT_IN_SATS ? royalty.amount : 0
-    this.outputs = [{ address: this.receiveAddress || this.address, value: this.price + this.postage - royaltyAmount }]
+    await this.calculateRoyalty()
```

Comment:
we can probably move this to the parent scope since it is not directly related to generating seller output? #nitpick

Reply:
Moved; https://github.com/sadoprotocol/ordit-sdk/pull/85/commits/74af66fe77ade668db586dd02a0013450346030e